aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -588,12 +605,22 @@ void gemm<at::BFloat16>(CUDABLAS_GEMM_ARGTYPES(at::BFloat16)) {
       c,
       CUDA_R_16BF,
       ldc,
+#if defined(USE_ROCM) && ROCM_VERSION >= 60000
+      CUBLAS_COMPUTE_32F,
```

Comment:
this is still hipblas api, are we supposed to use hipblaslt compute type here?

Reply:
In cuBLAS vs. cuBLASLt, I see the data types for compute type are the same:

https://docs.nvidia.com/cuda/cublas/#cublascomputetype-t

Ideally hipBLASLt should just follow this convension.

Reply:
@jianyuh in ROCm 6.0, the hipblaslt APIs are updated to align with cuBLAS/cuBLASLt w.r.t. types.  In the meantime, this PR tries to strike a balance between the state of the hipblaslt API in ROCm 5.7, hipblaslt develop branch, and hipblaslt 6.0 branch.  It is painful that the APIs changed so much, but if we can live with the workarounds in this PR for now that would be helpful.